web/app/services/build-log-websocket.service.ts

```diff
@@ -36,7 +40,7 @@ export class BuildLogWebsocketService {
   }
 
   private createSocketUrl(projectId: string, buildNumber: number) {
-    return `${this.API_ROOT}?project_id=${projectId}&build_number=${
-        buildNumber}`;
+    const token = this.authService.token();
```

Comment:
Note: We're going to be removing our own login and be using GitHub's authentication. So maybe here we can send GitHub's token and ensure they have access to the Build's target repo.

I'll take this as a note while working on the GitHub OAuth stuff.